<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Abdullahs-MBP.lan1" failures="1" tests="5" name="com.mercury.qa.testpages.LoginPageTest" time="28.750" errors="0" timestamp="03 Apr 2019 22:17:12 GMT" skipped="0">
  <testcase classname="com.mercury.qa.testpages.LoginPageTest" name="PageTitleValidation" time="3.814"/>
  <testcase classname="com.mercury.qa.testpages.LoginPageTest" name="loginpageTimeValidation" time="3.873"/>
  <testcase classname="com.mercury.qa.testpages.LoginPageTest" name="applicationVersion" time="4.620"/>
  <testcase classname="com.mercury.qa.testpages.LoginPageTest" name="loginvalidationwith_ValidUserName_and_ValidPassword" time="8.845"/>
  <testcase classname="com.mercury.qa.testpages.LoginPageTest" name="loginvalidationwithInvalidUserNameandValidPassword" time="7.598">
    <failure message="expected:&lt;Sign-on: Mercury Tours&gt; but was:&lt;Find a Flight: Mercury Tours:&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<Sign-on: Mercury Tours> but was:<Find a Flight: Mercury Tours:>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
at com.mercury.qa.testpages.LoginPageTest.loginvalidationwithInvalidUserNameandValidPassword(LoginPageTest.java:130)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginvalidationwithInvalidUserNameandValidPassword -->
</testsuite> <!-- com.mercury.qa.testpages.LoginPageTest -->
